client/coniksclient/internal/cmd/register.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"encoding/json"
+
+	"net/url"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+)
+
+var registerCmd = &cobra.Command{
+	Use:   "register",
+	Short: "Register a name-to-key binding.",
+	Long: `Register a new name-to-key binding on the CONIKS-server.
+
+Example call:
+  coniksclient register --name Alice@twitter --key fake_test_key`,
+	Run: func(cmd *cobra.Command, args []string) {
+		conf := loadConfigOrExit(cmd)
+		name := cmd.Flag("name").Value.String()
+		key := cmd.Flag("key").Value.String()
+		if len(name) == 0 || len(key) == 0 {
+			cmd.Usage()
+			return
+		}
+		req, err := createRegistrationMsg(name, key)
+		if err != nil {
+			fmt.Println("Couldn't marshal registration request!")
+			os.Exit(-1)
+		}
+
+		var res []byte
+		u, _ := url.Parse(conf.RegAddress)
```

Comment:
`conf.RegAddress` is defined with,

```
RegAddress string `toml:"registration_address,omitempty"`
```

Presumably this should fallback to `conf.Address` if empty.